apps/server/src/modules/collaborative-text-editor/domain/do/collaborative-text-editor.ts

```diff
@@ -0,0 +1,5 @@
+export interface CollaborativeTextEditor {
```

Comment:
This is in a folder called "do" but it is just an interface. Seems wrong.

Reply:
...or let's say it seems odd, because I don't know what's right in this case, as we don't have any consistent concept for those things.